components/molecules/HoverCardWrapper/hover-card-wrapper.tsx

```diff
@@ -25,7 +27,7 @@ const HoverCardWrapper = ({ username, repositories }: HoverCardWrapperProps) =>
   });
 
   const profile: ContributorsProfileType = {
-    githubAvatar: `https://www.github.com/${username}.png?size=60`,
+    githubAvatar: roundedImage(`https://www.github.com/${username}.png?size=60`, process.env.NEXT_PUBLIC_CLOUD_NAME),
```

Comment:
Why not use the `getAvatarById` function here?

Reply:
The id of the user is not available there. Most of the places where this component is used don't have the id of the user, only the name

Reply:
Ahh, I see